vllm/v1/worker/gpu_model_runner.py

```diff
@@ -157,9 +158,12 @@ def __init__(
         # Sampler
         self.sampler = Sampler()
 
-        # Lazy initialization
+        # Lazy initializations
         # self.model: nn.Module  # Set after load_model
+        # Initialize in initialize_kv_cache
         self.kv_caches: list[torch.Tensor] = []
+        self.kv_cache_config = cast(KVCacheConfig, None)
```

Comment:
Is it necessary to have this initialized as None? Can we just initialize it in `initialize_kv_cache`?